lib/normalizePath.js

```diff
@@ -17,7 +18,7 @@ export default function normalizePath(
   }
 
   if (
-    rawPath === '.' || rawPath.startsWith('./') || rawPath.startsWith('../')
+    rawPath === '.' || rawPath.startsWith('./') || rawPath.startsWith('../') || path.isAbsolute(rawPath)
```

Comment:
I tried this out locally, and it doesn't seem to have any effect. There's a line above that should handle absolute paths (starting with the working directory).